src/components/learner-credit-management/data/hooks/useBudgetDetailTabs.jsx

```diff
@@ -19,9 +19,10 @@ export const useBudgetDetailTabs = ({
   MembersTabElement,
 }) => {
   const intl = useIntl();
-  const isRetired = subsidyAccessPolicy?.retired;
   const showCatalog = (subsidyAccessPolicy?.groupAssociations?.length > 0 && !appliesToAllContexts)
   || (enterpriseFeatures.topDownAssignmentRealTimeLcm && !!subsidyAccessPolicy?.isAssignable);
+  const isEnded = !!subsidyAccessPolicy?.isEnded;
```

Comment:
[nit] If the original function is already determining truthiness of isEnded, there isn't a need to it again.

```js
function isSubsidyAccessPolicyEnded(policy) {
  return !!(policy.retired || !policy.isSubsidyActive);
}
```